src/config/utils.ts

```diff
@@ -113,3 +113,12 @@ function transformLoglevelToInt(loglevel: Loglevel): number {
       return 1;
   }
 }
+
+export function parseOptionalInt(
```

Comment:
This function should be covered by a unit test the same way `toArrayConfig` is…

Reply:
fixed